src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -163,6 +170,42 @@ public static FirebaseApp getInstance(@NonNull String name) {
     }
   }
 
+  /**
+   * Initializes the default {@link FirebaseApp} instance. Same as {@link
+   * #initializeApp(String)}, but it uses {@link #DEFAULT_APP_NAME} as name. *
+   * Also, it uses the {@link FirebaseOptions} values set in the file pointed to by
+   * {@link DEFAULT_CONFIG_OPTIONS_VAR}, if tht file is available.
+   * 
+   * <p>The creation of the default instance is automatically triggered at app startup time, if
+   * Firebase configuration values are available from resources - populated from
+   * google-services.json.
+   */
+  public static FirebaseApp initializeApp() {
+    return initializeApp(DEFAULT_APP_NAME);
+  }
+
+  /**
+   * Initializes the default {@link FirebaseApp} instance. Same as {@link
+   * #initializeApp(FirebaseOptions, String)}, but it uses the FirebaseOptions values set in the 
+   * file pointed to by {@link DEFAULT_CONFIG_OPTIONS_VAR}, if tht file is available.
+   * 
+   * <p>The creation of the default instance is automatically triggered at app startup time, if
+   * Firebase configuration values are available from resources - populated from
+   * google-services.json.
+   */
+  public static FirebaseApp initializeApp(String name) { 
+    DefaultConfigOptions defaultConfigOptions = DefaultConfigOptions.getOptionsFromEnv();
```

Comment:
Not a fan of the flow here. I'd recommend replacing this helper class with a helper method. See below.